pythesint/vocabulary.py

```diff
@@ -87,8 +87,8 @@ def sort_list(self, list):
             for _, key in enumerate(self.categories):
                 try:
                     line_kw[key] = dd[key]
-                except:
-                    line_kw = dd
+                except KeyError:
+                    continue
```

Comment:
I think it is tested. `MMDBaseVocabulary` also adds the keys `aboutCollection`, `prefLabelCollection` and  `definitionCollection` to the json file. Since these keys are not in the categories of the `mmd_*` fields in `pythesintrc.yaml`, the `KeyError` will be raised. Also, if you check the coveralls report, `vocabulary.py` should be fully covered.

Reply:
But I don't like that the coverage decreased...

Reply:
Coverage should also be solved now